### ISSUE TITLE
mobilenet-ssd download failure

### ISSUE THREAD

**Issue by ojjsaw** [openvinotoolkit/open_model_zoo#2774]
omz_downloader --name mobilenet-ssd

```
################|| Downloading mobilenet-ssd ||################

========== Downloading /data/obj-test/public/mobilenet-ssd/mobilenet-ssd.prototxt
... 100%, 28 KB, 135510 KB/s, 0 seconds passed

========== Downloading /data/obj-test/public/mobilenet-ssd/mobilenet-ssd.caffemodel
... 0%, 90 KB, 905 KB/s, 0 seconds passed
########## Error: Downloaded file is shorter (92211 B) than expected (23147564 B)

FAILED:
mobilenet-ssd

```

**Comment by vladimir-dudnik** [comment]
known issue https://github.com/openvinotoolkit/open_model_zoo/issues/2764

**Comment by vladimir-dudnik** [comment]
this gas been resolved with OpenVINO 2021.4.2 release, so I'm closing this issue
